inspirehep/config.py

```diff
@@ -334,6 +338,74 @@
     "980__a": ["collections.primary"],
     "980__b": ["collections.secondary"],
     "542__l": ["information_relating_to_copyright_status.copyright_status"],
+    
+    "037__c": ["arxiv_eprints.categories"],
+    "246__a": ["titles.title"],
+    "595": ["hidden_notes"],
+    "650__a": ["subject_terms.term"],
+    "695__a": ["thesaurus_terms.keyword"],
+    "773__y": ["publication_info.year"],
+    "anyfield": ["external_system_numbers.value",
```

Comment:
I don't think this fiels is needed. This was used in Legacy to implement Google-like (well not really) search, which is implemented in inspirehep/invenio-search#16